services/web-app/pages/designing/get-started.mdx

```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
   designTools={['Figma', 'Sketch', 'Adobe XD', 'Axure']}
```

Comment:
Can we add the caption text to each design tool? Figma can stay as is but we can link to the tutorials for the others. For example: 

`To learn more about installing Sketch Libraries visit the [Sketch tutorial](http://localhost:3000/designing/sketch).`